### ISSUE TITLE
people are reporting me

### ISSUE THREAD

**Issue by futi666** [finndev/PokeBuddy#643]
some people in my hometown are reporting me for bot using, someone knows about a permaban or something?

**Comment by curtiola** [comment]
I know it'll happen eventually but I don't know how seriously they take player reports, you should avoid telling these people the username of your next account lol

**Comment by futi666** [comment]
i was in a arena with lvl35, and then some of my friends say me that they are reporting me. 5-10 people

**Comment by curtiola** [comment]
Seem like good friends :/
Right now they don't have a report player feature that I know of, so I don't know if they can, and if they can somehow, Niantic will probably investigate before banning

**Comment by futi666** [comment]
iam hoping that iam not getting banned :/ thanks for ur help dude!

**Comment by TeemoOnShrooms** [comment]
You can report players, you can't tell exactly for what, but you can open tickets

**Comment by curtiola** [comment]
Ah, ty for letting me know

**Comment by futi666** [comment]
open tickets?

**Comment by curtiola** [comment]
What I gather from that is they can open tickets on the pokemon go website to report players

**Comment by futi666** [comment]
ah ok, thanks

**Comment by TeemoOnShrooms** [comment]
yeah if you go on the "Gear" (options) you can report bad issues: open ticket-> this guy cheat

**Comment by exclamati0n** [comment]
No offense but if you're botting why are you making it obvious to people that are concerned about such methods especially when you're concerned about your account?

You are your own "idiot beacon".

**Comment by 404unknown** [comment]
Dont fight gym with lvl30+ accounts, everyone will report you. Be smart, be safe!

**Comment by curtiola** [comment]
Yeah, just get the bot high level catch some super pokemon and wait for trading to be implemented xD

**Comment by IamDeric** [comment]
Just fight gym and leave it empty. Just get only the bonus when you win and someone will put a new defender on that gym, then destroy it again..
